IPCheck/dashAnalysis.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
+#*******************************************************************************
```

Comment:
You are right. I wanted to avoid managing too long paths but I will fix it. ANyhow this environment variable doesn't work well for me neither ;-).
Thx for your suggestion